jedi/evaluate/sys_path.py

```diff
@@ -33,8 +33,15 @@ def _get_venv_sitepackages(venv):
     if os.name == 'nt':
         p = os.path.join(venv, 'lib', 'site-packages')
     else:
-        p = os.path.join(venv, 'lib', 'python%d.%d' % sys.version_info[:2],
-                         'site-packages')
+        # Account for the VIRTUAL_ENV using an alternate python verison
+        venv_lib = os.path.join(venv, 'lib')
+        try:
+            pyver = [p for p in os.listdir(venv_lib) if
```

Comment:
I've meant to say that it should make sure to only return a single entry, probably not by using `startswith` (below, IIRC) / matching more strictly.